src/win.c

```diff
@@ -989,6 +986,24 @@ bool win_should_fade(session_t *ps, const struct managed_win *w) {
 	return ps->o.wintype_option[w->window_type].fade;
 }
 
+/**
+ * Determine if a window should animation.
+ */
+bool win_should_animation(session_t *ps, const struct managed_win *w) {
```

Comment:
Let's rename this to be proper english: `win_should_animation` -> `win_should_animate`

That being said, great call moving this to a function.